eng/tools/spec-gen-sdk-runner/test/utils/searchRelatedTypeSpecProjectBySharedLibrary.test.ts

```diff
@@ -10,14 +10,14 @@ describe("searchRelatedTypeSpecProjectBySharedLibrary", () => {
 
   test("finds related TypeSpec projects for shared libraries", () => {
     const sharedLibraries = [
-      path.normalize("specification/contosowidgetmanager/Contoso.WidgetManager.Shared/main.tsp"),
```

Comment:
Why remove this normalize call?

Reply:
In Windows,  the path.normalize("specification/contosowidgetmanager/Contoso.WidgetManager.Shared/main.tsp") will change to "specification\\contosowidgetmanager\\Contoso.WidgetManager.Shared\\main.tsp"